experimental/GModule/src/Cohomology.jl

```diff
@@ -193,6 +193,9 @@ function inv_action(C::GModule)
   return C.iac
 end
 
+# convert G-module into a matrix group
+Oscar.matrix_group(C::GModule) = matrix_group(matrix.(action(C)))
```

Comment:
this probably should(?) be more restricted: if the GModule operates on an abelian group, there won't be a matrix group. Don't know if shuld should be caught here (signature requiring modules over a field(?), checking for the module to be free in the current rep, ...)
Mind you: I am fine with trying (and possibly crashing)

Reply:
```suggestion
Oscar.matrix_group(C::GModule{<:Any, <:AbstractAlgebra.Module{<:FieldElem}}) = matrix_group(matrix.(action(C)))
```